bulk-load/src/main/java/no/sikt/nva/data/report/api/etl/loader/LoaderSpec.java

```diff
@@ -0,0 +1,99 @@
+package no.sikt.nva.data.report.api.etl.loader;
+
+import java.net.URI;
+import java.util.Locale;
+
+public record LoaderSpec(URI source,
+                         Format format,
+                         String iamRoleArn,
+                         String region,
+                         boolean failOnError,
+                         Parallelism parallelism,
+                         boolean updateSingleCardinalityProperties,
+                         boolean queueRequest) {
+
+    @Override
+    public String toString() {
+        return String.format("""
+                                 {"source": "%s",
+                                 "format": "%s",
+                                 "iamRoleArn": "%s",
+                                 "region": "%s",
+                                 "failOnError": "%s",
+                                 "parallelism": "%s",
+                                 "updateSingleCardinalityProperties": "%s",
+                                 "queueRequest": "%s",
+                                 "dependencies": []
+                                 }
+                                 """, source, format.toString(), iamRoleArn, region,
+                             booleanString(failOnError), parallelism.toString(),
+                             booleanString(updateSingleCardinalityProperties),
+                             booleanString(queueRequest));
+    }
+
+    private String booleanString(boolean value) {
+        var string = String.valueOf(value);
+        return string.toUpperCase(Locale.ROOT);
+    }
+
+    public static final class Builder {
+
+        private URI source;
+        private Format format;
+        private String iamRoleArn;
+        private String region;
+        private boolean failOnError;
+        private Parallelism parallelism;
+        private boolean updateSingleCardinalityProperties;
+        private boolean queueRequest;
+
+        public Builder() {
```

Comment:
I see that you use (in this project) mix of static builder() method and public Builder() constructor. Nice to use same pattern everywhere :)